src/main/kotlin/no/nav/familie/ef/personhendelse/kontantstøtte/vedtak/KontantstøtteVedtakListener.kt

```diff
@@ -0,0 +1,67 @@
+package no.nav.familie.ef.personhendelse.kontantstøtte.vedtak
+
+import com.fasterxml.jackson.module.kotlin.readValue
+import no.nav.familie.eksterne.kontrakter.VedtakDVH
+import no.nav.familie.kontrakter.felles.objectMapper
+import org.apache.kafka.clients.consumer.ConsumerRecord
+import org.slf4j.LoggerFactory
+import org.springframework.kafka.annotation.KafkaListener
+import org.springframework.kafka.listener.ConsumerSeekAware
+import org.springframework.stereotype.Component
+
+@Component
+class KontantstøtteVedtakListener(val kontantstøtteVedtakService: KontantstøtteVedtakService) : ConsumerSeekAware {
+
+    private val logger = LoggerFactory.getLogger(javaClass)
+    private val securelogger = LoggerFactory.getLogger("secureLogger")
+    private val vurderKonsekvensOppgaveBeskrivelse =
+        "Bruker har fått vedtak om kontantstøtte og har løpende barnetilsyn"
+
+    @KafkaListener(
+        id = "aapen-kontantstotte-vedtak-aiven",
+        groupId = "familie-ef-kontantstøtte-vedtak",
+        topics = ["\${FAMILIE_KS_VEDTAK_TOPIC}"],
+        containerFactory = "kafkaKontantstøtteVedtakListenerContainerFactory",
+    )
+    fun listen(consumerRecord: ConsumerRecord<String, String>) {
+        val vedtakhendelse = objectMapper.readValue<VedtakDVH>(consumerRecord.value())
```

Comment:
De vedtakene som KS gjør - er det bare innvilgelse, eller får vi avslag/opphør her også? Hvis sistnevnte - er det noen måte å sile vekk avslag/opphør på?

Reply:
Det er bare innvilgelser, så vi trenger ikke å sjekke for avslag, etc

Reply:
Resolver denne da vi ble enige om å droppe sjekk for opphør siden det gjelder så få tilfeller 👍